examples/complete/README.md

```diff
@@ -0,0 +1,48 @@
+# atlas-basic - complete
+
+This example shows how the atlas-basic module can be used to create a MongoDB Atlas project, an AWS cluster for the project, several IP addresses and CIDR blocks for the project, and several database users. 
```

Comment:
would a bullet list work better here? 
Also consider replacing `AWS cluster` with MongoDB Atlas cluster provisioned with AWS provider to avoid confusion.